modules/jtframe/hdl/ver/test.cpp

```diff
@@ -695,7 +695,7 @@ void JTSim::reset( int v ) {
 }
 
 JTSim::JTSim( UUT& g, int argc, char *argv[]) :
-    game(g), sdram(g), dwn(g), sim_inputs(g), wav("test.wav",48000,false)
+    wav("test.wav",48000,false), sdram(g), sim_inputs(g), dwn(g), game(g)
```

Comment:
Was this order change needed?

Reply:
gcc 14.1 errors out if the member order and the initialization order differs.